styles/index.css

```diff
@@ -13,6 +13,16 @@ body {
   background-color: var(--white);
 }
 
+img {
```

Comment:
@hitomipupil  sorry for late comment - but i would suggest you to assign class name for css styling. since it can impact other images on the pages 
or assign this as `main img {}`.